commands/new_function.go

```diff
@@ -99,6 +102,22 @@ func runNewFunction(cmd *cobra.Command, args []string) error {
 		if _, statErr := os.Stat(appendFile); statErr != nil {
 			return fmt.Errorf("unable to find file: %s - %s", appendFile, statErr.Error())
 		}
+
+		originalBytes, readErr = ioutil.ReadFile(appendFile)
+		if readErr != nil {
+			fmt.Printf("unable to read %s to append, %s", appendFile, readErr)
+		}
+
+		services, parseErr := stack.ParseYAMLData(originalBytes, "", "")
```

Comment:
I'd like to see these 3-4 statements which pertain to duplicate-checking extracted into a new local/private function.